scripts/package/package_xcode_and_sdks.sh

```diff
@@ -84,7 +90,12 @@ function package_xcode() {
 
 	pushd ${XCODE}
 
-	EXTRA_ARGS="--exclude=${_name}/Developer/Platforms --exclude=${_name}/usr/lib/sourcekitd.framework --exclude=${_name}/usr/metal"
+	EXTRA_ARGS="--exclude=${_name}/usr/bin --exclude=${_name}/Developer/Platforms --exclude=${_name}/usr/lib/sourcekitd.framework --exclude=${_name}/usr/metal"
```

Comment:
Making a slimmed down version for the extender

Reply:
I should however create two packages "original" + "slim", since we need them both in different scenarios.
Original, when building on osx (until we use a vanilla clang version)
Slim, for the extender server